src/DurableTask.SqlServer.AzureFunctions/SqlScaleMonitor.cs

```diff
@@ -23,14 +23,13 @@ class SqlScaleMonitor : IScaleMonitor<SqlScaleMetric>
 
         int? previousWorkerCount = -1;
 
-        public SqlScaleMonitor(string taskHubName, SqlMetricsProvider sqlMetricsProvider)
+        public SqlScaleMonitor(string functionId, string taskHubName, SqlMetricsProvider sqlMetricsProvider)
         {
-            // Scalers in Durable Functions are shared for all functions in the same task hub.
-            // So instead of using a function ID, we use the task hub name as the basis for the descriptor ID.
+            // Scalers in Durable Functions is per function ids. And scalers share the same sqlMetricsProvider in the same taskhub. 
             string id = $"DurableTask-SqlServer:{taskHubName ?? "default"}";
```

Comment:
string id = $"{functionId}-DurableTask-SqlServer:{taskHubName ?? "default"}";
```suggestion
            string id = $"DurableTask-SqlServer:{taskHubName ?? "default"}";
```

Reply:
Thanks! does this string id needs to add ToLower(CultureInfo.InvariantCulture) to be lower case?